dev/storage.h

```diff
@@ -865,11 +865,11 @@ namespace sqlite_orm {
              *  @return id of just created object.
              */
             template<class O>
-            int insert(const O& o) {
+            int64 insert(const O& o) {
```

Comment:
I'd leave it like that cause changing return type of public API which exists 6+ years will spawn a lot of warnings. `int` is the most popular type and the first type every developer uses in the first C++ program. And also for 7+ years nobody complained about `int` here. Who needs the highest precision just uses `insert` with `last_insert_rowid` calls together. Now one person decided it is wrong and we are adding a lot of warning to all other projects. This is a bad practice. The better solution is to leave this public API as is and tell that one person what to do instead.